cpp/src/io/comp/gpuinflate.cu

```diff
@@ -1065,6 +1054,7 @@ CUDF_KERNEL void __launch_bounds__(block_size)
   __syncthreads();
   // Main loop decoding blocks
   while (!state->err) {
+    __syncthreads();
```

Comment:
racecheck fix - state->err is read by all threads on line 1056, and thread zero potentially changes the value in the code below.